src/04-modality-specific-files/09-positron-emission-tomography.md

```diff
@@ -276,6 +276,27 @@ and a guide for using macros can be found at
 
 All reconstruction-specific parameters that are not specified, but one wants to include, should go into the `ReconMethodParameterValues` field.
 
+#### Task
+
+If the OPTIONAL [`task-<label>`](../99-appendices/09-entities.md#task) is used,
```

Comment:
What does not having `task` imply? Just that the scan is resting-state, or something more specific? Like it's meant as a static, rather than functional, scan or something?

Reply:
Given that it's PET it could just be that you are measuring binding potential of some ligand I suspect. Some not exactly static because you have a temporal component but not functional for sure. @mnoergaard ?

Reply:
Not having a task means rest (to lie still in the scanner and note move), and this is how most of PET scans are carried out. Both for static and dynamic scans.